apps/fares/lib/fare_info.ex

```diff
@@ -5,7 +5,7 @@ defmodule Fares.FareInfo do
 
   alias Fares.Fare
 
-  @july_1_2019 1_561_953_600
+  @july_1_2019 1_561_939_199
```

Comment:
1561953600 converts to Monday July 01, 2019 00:00:00 (am) in time zone America/New York (EDT)
https://www.epochconverter.com/timezones?q=1561953600&tz=America%2FNew_York

🤔

Reply:
It probably doesn't matter too much, but the fares will take effect ~3am.

Reply:
^👍 

I think it'd make more sense to the average user to have the fares on the site switch at 12am. If someone ends up paying less at 1am, I can't imagine they'd complain. I do think we could educate riders about the 3am service day but not within this work.

The changed time stamp 1561939199 converts to Sunday June 30, 2019 19:59:59 (pm) in time zone America/New York (EDT) so that's early.

Reply:
I'm going to leave this alone for now. Will change at midnight EDT per Dave Barker's request.